locales/nl_NL.json

```diff
@@ -1,109 +1,109 @@
 {
   "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
-  "buy": "buy",
-  "sell": "sell",
-  "continue": "Continue",
+  "buy": "koop",
+  "sell": "verkoop",
+  "continue": "verder",
```

Comment:
Ik denk inderdaad dat zoals je in lijn 63 beschreef "Ga verder" het meest toepasselijk is.

Reply:
Idd, thx!